source/_posts/2022-09-30-php-core-roundup-6.md

```diff
@@ -0,0 +1,290 @@
+  url: https://aye.sh
+published_at: 30 September 2022
+---
+
+Welcome back to _PHP Core Roundup_, the sixth in the [series](https://thephp.foundation/blog/tag/roundup/). _PHP Core Roundup_ summarizes the latest improvements, bug fixes, discussions, and proposals to PHP.
+
+> The PHP Foundation currently supports six part-time PHP contributors who work on maintenance and new features for PHP. Maintenance is not limited to fixing bugs, but also includes work to reduce technical debt, making life easier for everyone working on PHP. The contributors funded by the PHP Foundation collaborate with other contributors on code, documentation, and discussions.
+
+<br>
+We publish the posts on our website, and you can subscribe to a newsletter; You don’t necessarily have to be a PHP Foundation backer to follow _PHP Core Roundup_.
+
+<div class="px-4 pt-3 pb-10 mb-6 border-b border-t -mx-4 border-gray-200">
+    <div class="max-w-xl mx-auto">
+        <h2 class="text-xl text-left inline-block font-semibold text-gray-800 mb-1">Subscribe to PHP Core Roundup newsletter</h2>
+        <form method="POST" action="https://php-foundation-mailcoach.com/subscribe/9be4e2bd-f9d8-475c-b00e-2dcc4cf90056" class="mt-2">
+            <div class="flex items-center">
+                <input placeholder="Your email address" type="email" class="w-full px-2 py-4 mr-2  bg-gray-100 shadow-inner rounded-md border border-gray-400 focus:outline-none" name="email" required>
+                <button class="bg-[#7f52ff] text-gray-200 px-5 py-2 rounded shadow " style="margin-left: -7.8rem;">Sign Up</button>
+            </div>
+        </form>
+    </div>
+</div>
+
+
+September was a slow month, given that PHP 8.2 reached its feature-freeze, and most of the work is now focused on getting PHP 8.2 ready for its Generally Available releases, with PHP 8.2.0 scheduled for November 24th.
+
+[Nikita Popov](https://github.com/nikic) recently tagged three new releases of the [nikic/PHP-Parser](https://github.com/nikic/PHP-Parser) library with support for PHP 8.2 features and syntax. PHP-Parser is used as the underlying PHP parser for several PHP static analyzers, and you can expect many PHP tools to support new PHP 8.2 syntax in the coming weeks. 
+
+
+## Early-developments for PHP 8.3
+
+Although PHP 8.2 is still being ironed out, there are some discussions and even an RFC currently being voted for proposed changes in PHP 8.3 (scheduled for the end of 2023!).
+
+[Deprecations for PHP 8.3 RFC](https://wiki.php.net/rfc/deprecations_php_8_3) is currently in draft, which currently stands to deprecate certain features related to the `mb_strimwidth` function, `NumberFormatter` class, and a few functions related to Random Number Generators (RNG).
+
+Furthermore, there is an [RFC](https://wiki.php.net/rfc/json_validate) currently under vote to add a new function named _json_validate_, that validates a given JSON string without the memory overhead of otherwise decoding the JSON string into memory. 
```

Comment:
```suggestion
Furthermore, there is an [RFC](https://wiki.php.net/rfc/json_validate) currently under vote to add a new function named `json_validate`, that validates a given JSON string without the memory overhead of otherwise decoding the JSON string into memory. 
```

Reply:
Thank you, added.